__tests__/app.test.js

```diff
@@ -138,4 +137,49 @@ describe('GET: /api/articles/:article_id/comments', () => {
             expect(msg).toBe("Bad request")
         })
     })
+});
+describe('POST: /api/articles/:article_id/comments', () => {
+    it('201: responds with correct status and adds a comment object to article with correct properties (username and body)', () => {
```

Comment:
We should add a test for when there are unnecessary properties in the comment I.E. {fruit: 'apple', username: 'butter_bridge', body: 'this is a comment'}. We want to ignore the extra keys and send back 201 and the comment 🙂